counterpartylib/lib/config.py

```diff
@@ -7,7 +7,7 @@
 # Versions
 VERSION_MAJOR = 9
 VERSION_MINOR = 55
-VERSION_REVISION = 1
+VERSION_REVISION = 2
```

Comment:
I think a protocol change should require a version bump but I never really thought about the auto-triggered reparse being unwanted before